parktain/main.py

```diff
@@ -65,12 +65,23 @@ def checkins_reminder():
     date = datetime.now().strftime('%d %B, %Y')
     bot.speak('Morning! What are you doing on {}!'.format(date), "#checkins")
 
-@bot.listen_for(lambda user, channel, message: True)
+@bot.listen_for(lambda user, channel, message: True, target_channel='clickbaits')
 def logger(user, channel, message):
     message_log = Message(user_id=user, channel_id=channel, message=message, timestamp=datetime.now())
     session.add(message_log)
     session.commit()
 
+    # Check for presence of HyperLink in message
```

Comment:
I'm leaning towards having a separate handler for this. Just keep it simple and let one function do one thing?